packages/open-ui/src/components/Tag/Tag.module.scss

```diff
@@ -0,0 +1,16 @@
+@import 'styles/index.scss';
+
+.Tag {
+  background-color: $grey;
+  border-radius: 12px;
+  color: $black;
+  display: inline-block;
+  font-size: $font-size-sm;
+  padding: $vr;
+  text-transform: uppercase;
+}
+.large {
+  padding: $vr * 1.5;
+  border-radius: 20px;
```

Comment:
For rounded corners pls use `border-radius: 50%`;

Reply:
@remotealex I already tried it I believe, they were creating too much of roundness